### PR TITLE
Skill related improvements

### PR DESCRIPTION
**Pull request by mikhail-o**
Added handling client packet 3F (RequestSkillList)
Fixed loading skilldb_edit.dlc (loading skills to memory)

### CODE REVIEW

File: src/L2dotNET/model/skills2/SkillTable.cs [hunk 1]
```diff
@@ -465,8 +468,12 @@ private void LoadDlc()
 
                     list.Skills.Add(skill);
                 }
+                if (list.Type != "")
```
Comment (Elfocrash) [inline src/L2dotNET/model/skills2/SkillTable.cs]: use !string.IsNullOrEmpty

File: src/L2dotNET/model/skills2/SkillTable.cs [hunk 2]
```diff
@@ -184,9 +185,10 @@ public void Read()
                                     value = dlc.ReadS(lenx);
                                     try
                                     {
-                                        skill.TargetType = (SkillTarget)Enum.Parse(typeof(SkillTarget), value);
+                                        var newValue = StringHelper.ToTitleCase(value, '_');
+                                        skill.TargetType = (SkillTarget)Enum.Parse(typeof(SkillTarget), newValue);
                                     }
-                                    catch (Exception)
+                                    catch (Exception ex)
```
Comment (Elfocrash) [inline src/L2dotNET/model/skills2/SkillTable.cs]: Add ex to the log

File: src/L2dotNET.Repositories/SkillRepository.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        public SkillRepository()
+        {
+            Db = new MySqlConnection(ConfigurationManager.ConnectionStrings["PrimaryConnection"].ToString());
+        }
+
+        public List<SkillResponseModel> GetPlayerSkills(int charID)
+        {
+            try
+            {
+                const string sql = @"select ownerId as CharObjId, id as skillId, lvl as SkillLvl, iclass as ClassId from user_skills where ownerId = @char_obj_id";
```
Comment (Elfocrash) [inline src/L2dotNET.Repositories/SkillRepository.cs]: change const string to var

**Comment by mikhail-o** [comment]
Hi
Did all changes, if there is something else, let me know.

**Comment by Elfocrash** [comment]
@mikhail-o Looks good. Thanks and welcome 🥇

**Pull request merged**